website/docs/feature-flag-tutorials/dotnet/dotnet-examples.md

```diff
@@ -0,0 +1,397 @@
+
+Your release process is now configured with an environment-dependent strategy constraint. Since we've set the rollout to 100%, the feature will be released to all users that are not in the `production` environment.
+
+Alternatively, you can send an API command to apply the same requirements:
+
+```csharp
+HttpClient client = new HttpClient();
+
+string url = "<your-unleash-url>/api/admin/projects/:projectId/features/:featureName/environments/:environment/strategies/:strategyId";
+
+var payload = new
+{
+    name = "flexibleRollout",
+    disabled = false,
+    constraints = new[]
+    {
+        new
+        {
+            values = new[] { "production" },
+            inverted = false,
+            @operator = "NOT_IN",
+            contextName = "environment",
+            caseInsensitive = false
+        }
+    },
+    variants = new object[] { },
+    parameters = new
+    {
+        groupId = "delete_survey_flag",
+        rollout = "50",
+        stickiness = "sessionId"
+    },
+    segments = new object[] { }
+};
+
+client.DefaultRequestHeaders.Add("Accept", "application/json");
+client.DefaultRequestHeaders.Add("Authorization", "<API_KEY_VALUE>");
+
+var response = await client.PutAsJsonAsync(url, payload);
+var responseBody = await response.Content.ReadAsStringAsync();
+
+Console.WriteLine(responseBody);
+```
+
+> Note:
+> If you already have a gradual rollout strategy for your flag, use a PUT request to update it.
+> If you’re creating a new strategy, use a POST request.
+
+Check out our [API docs on updating flag strategies](/reference/api/unleash/update-feature-strategy) to learn more.
+
+Read our documentation for more context on [strategy constraint configurations](/reference/strategy-constraints) and use cases.
+
+## Server-side A/B Testing in .NET
+
+A/B testing is a common way for teams to test out how users interact with two or more versions of a new feature that is released. At Unleash, we call these [strategy variants](/reference/strategy-variants).
+
+When a feature flag is enabled, we can expose a particular version of a feature to a select user base. From there, we can use the variants to view the performance metrics in Unleash and see which is more efficient.
+
+In Unleash, navigate go to a feature flag, then click on an environment to open your strategy.
+
+![Open the strategy menu](../ruby/strategy.png)
+
+Click ‘Edit Strategy'
+
+![Add a variant to your gradual rollout strategy.](/img/ex-add-strategy-variant.png)
+
+Only enable your flag for 50% of users.
+
+![The slider that controls the graduality of the rollout is set to 50%](../ruby/50-gradual.png)
+
+Alternatively, you can do that with a `PUT` request in .NET using our API:
+
+```csharp
+HttpClient client = new HttpClient();
+
+string url = "<your-unleash-url>/api/admin/projects/<your-project-id>/features/<your-feature-flag>/environments/<your-environment>/strategies/3a76899f-582b-422f-be72-34c995388f77";
```

Comment:
Same here and for the urls further down. and you've accidentally included a reference to one of your own strategies here?